terraform/gitops/pm4ml/pm4ml.tf

```diff
@@ -59,7 +59,7 @@ module "generate_pm4ml_files" {
     pm4ml_external_switch_oidc_token_route          = each.value.pm4ml_external_switch_oidc_token_route
     pm4ml_external_switch_client_secret             = var.pm4ml_external_switch_client_secret
     pm4ml_external_switch_client_secret_key         = "token"
-    pm4ml_external_switch_client_secret_vault_key   = "${var.kv_path}/${var.cluster_name}/${each.key}/${each.value.pm4ml_external_switch_client_secret_vault_path}"
+    pm4ml_external_switch_client_secret_vault_key   = "${var.cluster_name}/${each.key}/${each.value.pm4ml_external_switch_client_secret_vault_path}"
```

Comment:
Why cluster_name is removed? I think we are accessing tenancy vault and there might be many clusters in that.